app/services/document_capture_session_async_result.rb

```diff
@@ -22,6 +22,8 @@ def done?
     status == DocumentCaptureSessionAsyncResult::DONE
   end
 
+  alias_method :success?, :done?
```

Comment:
reason this is needed: the capture doc flow expecs to load a `DocumentCaptureSessionResult` which is basically the same as this one, except that it responds to `success?`